src/content_handlers/CMakeLists.txt

```diff
@@ -38,6 +38,8 @@ add_subdirectory (swupdate_handler)
 #   add_subdirectory (my_handler)
 #
 
+target_compile_definitions (${PROJECT_NAME} PUBLIC ADUSHELL_FILE_PATH="${ADUSHELL_FILE_PATH}")
```

Comment:
apt_handler.cpp also uses `adushconst::adu_shell` and `#include`'s `adushell_const.hpp`, so I would expect this `target_compile_definitions` to be in `src/content_handlers/apt_handler/CMakeLists.txt` also.

Reply:
Done.